demo/index.ts

```diff
@@ -51,8 +51,20 @@ const {
   pullquoteElement,
 });
 
+const restrictedParagraph = {
+  content: "text*",
+  marks: "em",
```

Comment:
What is this additional node spec being used for? It looks superfluous to me,  unless I've missed something.

Reply:
Great catch – this was used when me and @rhystmills were proving the concept. It's no longer needed.